src/derivative_operators/coefficients.jl

```diff
@@ -0,0 +1,32 @@
+"""
+```
+init_coefficients(coeff_func, len::Int)
+```
+
+Return the initial value of an operator's `coefficients` field based on the type
+of `coeff_func`.
+"""
+init_coefficients(coeff_func::Nothing, len::Int) = nothing
+
+init_coefficients(coeff_func::Number, len::Int) = coeff_func * ones(typeof(coeff_func), len)
+
+function init_coefficients(coeff_func::AbstractVector{T}, len::Int) where T <: Number
+    coeff_func
+end
+
+init_coefficients(coeff_func::Function, len::Int) = ones(Float64, len)
+
+
+
+"""
+```
+get_coefficient(coefficients, index)
+```
+"""
+get_coefficient(coefficients::AbstractVector, index::Int) = coeff[i]
+
+# FIXME: I don't think this case is used anymore
```

Comment:
This case was covered by a subset of convolution functions, but I don't know what the use-case is.

Reply:
It's for a constant coefficient, like D*A so a constant diffusion. If the coefficient is a number, then the whole setup is actually a bitstype so it'll compile to the GPU and distribute much better, and this is a common case so it's worth supporting. I think that one function is really all that's necessary to support it.